src/components/markdown/allowed-css-properties.ts

```diff
@@ -3,10 +3,12 @@ export const allowedCssProperties = [
     'color',
     'font-style',
     'font-weight',
+    'height',
     'text-decoration-color',
     'text-decoration-line',
     'text-decoration-skip-ink',
     'text-decoration-style',
     'text-decoration-thickness',
     'text-decoration',
+    'width',
```

Comment:
I didn't test. But does it mean it works on both of these, inside the `style` tag and if they are directly tags?
```
    width="31"
    height="31"
    style="width: 31px; height: 31px;"
```

Reply:
Ah, good point!

No, it doesn't. This change only adds them as allowed properties inside the `style` attribute. I just checked, and `height` and `width` added as properties are filtered out. I'll add a fixup!

Reply:
⚡